src/WorkItemFiling/WorkItemFiling.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <RootNamespace>Microsoft.WorkItemFiling</RootNamespace>
+  </PropertyGroup>
+
+  <PropertyGroup Label="AssemblyAttributes">
+    <AssemblyTitle>Universal Work Item Filing Library</AssemblyTitle>
+    <Description>Classes for filing work items in either GitHub or AzureDevOps.</Description>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Json.Schema" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Json.Schema.Validation" Version="1.1.1" />
+    <PackageReference Include="Microsoft.VisualStudio.Services.Client" Version="16.143.2" />
```

Comment:
>16.143.2 [](start = 80, length = 8)

Not sure if we want to do an update now, but the latest version 16.153.2 was released several months ago.

Reply:
good point! will do this next.

---
In reply to: [377353664](https://github.com/microsoft/sarif-sdk/pull/1749#discussion_r377353664) [](ancestors = 377353664)